lips/lip-10.md

```diff
@@ -73,6 +75,32 @@ struct LibraIdDomain {
 * The Libra ID domain can be created as part of creating a `ParentVASP` account.
 * An entity (potentially the Association) may at some point choose to expose an open source application(*Indexer*) that will allow indexing available Libra IDs and could provide a convenient REST API for applications to fetch information about the domains. The API of such an indexer is out of scope of this RFC.
 
+## Libra ID domain events
+
+Blockchain contract that manages Libra ID domains must emit an event every time Libra ID domain is created, removed or updated. Those events are critical for applications to be able to efficiently index existing Libra ID domains.
+
+While Libra ID domains published into VASP accounts, Libra ID domain events are published under the Treasury Compliance account. We consolidate events under single account to allow indexers to follow single event stream.
+
+To store events, `LibraIdDomainManager` resource is published under the Treasure Compliance account(address `0xB1E55ED`).
+
+```
+resource struct LibraIdDomainManager {
+    /// Events are emmitted 
+    libra_id_domain_events: Event::EventHandle<Self::LibraIdDomainEvent>,
+}
+
+struct LibraIdDomainEvent {
+    removed: bool,
+    domain: LibraIdDomain,
+    address: address,
+}
+```  
+
+* Fields definition:
+  * `removed` - `true`, if LibraIdDomain was removed, `false` if it was created or updated
+  * `domain` - exact copy of `LibraIdDomain` that was added/removed from `LibraIdDomains` resource of a VASP account
+  * `address` - address of an account where `LibraIdDomain` was added or removed
```

Comment:
Just to clarify here the address here is a parent Vasp address?

Reply:
Yes

Reply:
I had the same question as Lu.   Can we add that this is the parent address and example values also?

Reply:
What does `modified` mean?